Add Logout popup

timmy3000 opened the pull request:
i have added a logout popup button, it can be accessed by clicking on the avatar.

timmy3000 committed:
Add Logout popup

PhilemonBrain commented:
@timmy3000 Thanks for this PR, but i cant even seem to view the dashboard, there are a couple of errors on the console, please check it out.

timmy3000 committed:
Update Menus.tsx

added "const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);"
